ui/src/apps/overview/RootComponent.js

```diff
@@ -6,20 +6,49 @@ import client from '@pingcap-incubator/dashboard_client'
 
 import { ComponentPanel, MonitorAlertBar } from './components'
 import styles from './RootComponent.module.less'
+import { StatementsTable } from '@pingcap-incubator/statement'
+import moment from 'moment'
+
+const timeFormat = 'YYYY-MM-DD HH:mm:ss'
 
 const App = () => {
   const [cluster, setCluster] = useState(null)
+  const [topStatements, setTopStatements] = useState([])
+  const [loadingStatements, setLoadingStatements] = useState(false)
   const { t } = useTranslation()
+  const [timeRange] = useState(() => {
```

Comment:
how about just use the latest available period?

Reply:
Good idea!